src/controllers/docker.js

```diff
@@ -393,11 +393,23 @@ class Docker {
             this.server.log.debug('Removing old server container...');
             // @TODO: logic here to determine if this failed, if so we need to
```

Comment:
Well it seemed to describe something else so i kept it. If you meant the check i implemented i can remove it.